index.html

```diff
@@ -8,23 +8,25 @@
   <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <header class="">
-    <h1 class="">App de descuentos</h1>
+  <header class="header-title">
```

Comment:
recuerda usar BEM te dejo la sugerencia:
```suggestion
  <header class="header__title">
```